### PR TITLE
Add sunset notification to popup UI

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
This PR:
- Adds messaging about the FPN sunset to all panels of the FPN extension 
- Changes the badge icon to a persistent warning icon 
- Updates the version number to "25.0.0", though this is overwritten somewhere in TaskCluster
- Stops redirecting folks to the exit survey when the extension is uninstalled 
- Still negotiating with the tests on my M1. This updates one test but is possibly failing others.

### CODE REVIEW

File: src/popup/popup.css
Comment (pdehaan) [inline src/popup/popup.css:666]: nit: merge these?
```suggestion
.sunset {
  background-color: var(--warning);
  display: inline;
}
```

Comment (lesleyjanenorton) [reply src/popup/popup.css:666]: Well that's embarrassing

Comment (pdehaan) [reply src/popup/popup.css]: /giphy "Shame!"

File: test/extension.js
Comment (pdehaan) [inline test/extension.js:153]: Q: Is the badge permanently "badge_warning.svg" now and do we need to delete other unused SVGs?
Or, if the whole repo is archiving (in mid-June, per a messages.json edit above), do we care?

File: src/background/ui.js
Comment (pdehaan) [inline src/background/ui.js:212]: Not sure this needs `Promise.all()` anymore since it's just one function now. But removing it might also convert it from a Promise to something non-promisey. But then again the function seems to be `async` so probably a Promise regardless.

**Review by pdehaan — COMMENTED** [review]
I didn't build+run this locally, but code "looks" good. 
I can blind R+ if needed, as long as we have Softvision or somebody that can install the add-on and run the gauntlet of tests (Windows, High Contrast, etc).

File: test/extension.js
Comment (lesleyjanenorton) [reply test/extension.js:153]: The whole repo will be archived. Happy to remove them though in the interest of Most Correctness.

File: src/background/ui.js
Comment (lesleyjanenorton) [reply src/background/ui.js]: oof. updated.

**Comment by lesleyjanenorton** [comment]
@pdehaan Softvision will indeed be running the guantlet so if you're comfortable blind R+ing that would be fantastic although actually not entirely necessary since they can use the TaskCluster builds triggered by this PR and locally built XPIs. We're not blocked on merging so your call!

**Review by pdehaan — APPROVED** [review]
:yolo:

**Pull request merged**
